src/main/java/taxi/controller/web/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,46 @@
+package taxi.controller.web.filter;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class AuthenticationFilter implements Filter {
+    private Set<String> allowedUrls = new HashSet<>();
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+        allowedUrls.add("/login");
+        allowedUrls.add("/register");
+        allowedUrls.add("/index");
```

Comment:
this is redundant 
<img width="708" alt="image" src="https://user-images.githubusercontent.com/61326568/191514711-961f7371-b388-4b50-91ec-8bc4938bb3c5.png">
```suggestion
```

Reply:
yes, i just forget to remove this/ we were testing one case with Sophia with this